website/content/docs/enterprise/licensing.mdx

```diff
@@ -5,52 +5,103 @@ description: >-
 
 The Boundary Enterprise binaries require a license key to enforce entitlements and time restrictions according to your contractual agreement.
 Once purchased, your HashiCorp account team send you a valid Boundary license for environment activation.
 If you don't yet have an acccount team, but are interested in Boundary Enterprise, you can request access [here](https://www.hashicorp.com/contact-sales).
 
+## Enable Boundary Enterprise
+
 Once you have received your valid Boundary license, there are two options for enabling Boundary Enterprise:
 
 1. You may set the enterprise license for Boundary using an environment variable.
 Set the `BOUNDARY_LICENSE` environment variable to the license key value you received:
 
-   `$ export BOUNDARY_LICENSE=02MV4UU43BK5...`
+  `$ export BOUNDARY_LICENSE=02MV4UU43BK5...`
 
-   or
+  or
 
-   `$ BOUNDARY_LICENSE=file:///folder_path/license.hclic`
+  `$ BOUNDARY_LICENSE=file:///folder_path/license.hclic`
 
 1. You may specify the enterprise license key in the Boundary controller configuration using the `license` attribute in the `controller` block.
 You can either specify the raw license key or the path to a file containing the license key.
 Refer to the sample configurations below:
 
    **Sample #1**
 
-   ```hcl
-   controller {
-     name = "controller"
-     description = "controller"
-     database {
-       url = "postgresql://boundary:PASSWORD@127.0.0.1:5432/boundary"
-     }
-     public_cluster_addr = "boundary.domain"
-     license = "file:///folder_path/license.hclic"
-   }
-   ```
+  ```hcl
+  controller {
+    name = "controller"
+    description = "controller"
+    database {
+      url = "postgresql://boundary:PASSWORD@127.0.0.1:5432/boundary"
+    }
+    public_cluster_addr = "boundary.domain"
+    license = "file:///folder_path/license.hclic"
+  }
+  ```
    **Sample #2**
 
-   ```hcl
-   controller {
-     name = "controller"
-     description = "controller"
-     database {
-       url = "postgresql://boundary:PASSWORD@127.0.0.1:5432/boundary"
-     }
-     public_cluster_addr = "boundary.domain"
-     license = "02MV4UU43BK5..."
-   }
-   ```
-
-   Refer to the [controller](/boundary/docs/configuration/controller) configuration  documentation for more information.
\ No newline at end of file
+  ```hcl
+  controller {
+    name = "controller"
+    description = "controller"
+    database {
+      url = "postgresql://boundary:PASSWORD@127.0.0.1:5432/boundary"
+    }
+    public_cluster_addr = "boundary.domain"
+    license = "02MV4UU43BK5..."
+  }
+  ```
+
+  Refer to the [controller](/boundary/docs/configuration/controller) configuration  documentation for more information.
+
+## License expiration
+
+When the Boundary license expires, the controller performs a graceful shutdown.
+Ongoing sessions continue to run, but you will not be able to create any new sessions.
+
+Prior to the expiration date, Boundary logs 5 warnings to the event stream at the following intervals:
```

Comment:
Are you able to provide an example of what that log looks like? That would be helpful for users to configure alerting on this, just in case.